pdns/recursordist/docs/appendices/FAQ.rst

```diff
@@ -0,0 +1,44 @@
+Frequently Asked Questions
+==========================
+
+This document lists categorized answers and questions with links to the relevant documentation.
+
+EDNS bufsize in response packets
+--------------------------------
+
+You may have spotted the 512 in something like the following (after ``EDNS ... udp:``)::
+
+  $ dig example.com @9.9.9.9
```

Comment:
maybe `@127.0.0.1` or `192.0.2.2`?

Reply:
makes sense, I wanted to keep it real but you're probably right :)